tests/test_clients_GcpClient.py

```diff
@@ -748,6 +751,47 @@ def test_download_from_blobstore_raises_exception_on_invalid_container(self):
             Exception, self.gcpClient._download_from_blobstore, 'blob', invalid_blob_path)
         self.gcpClient.container = prev_container
 
+    @patch('lib.clients.BaseClient.requests.post')
+    @patch('lib.clients.BaseClient.requests.get')
```

Comment:
Let's try to keep same patching method throughout the class. Can you please check if you can make it similar to how it is done [here](https://github.com/cloudfoundry-incubator/service-fabrik-backup-restore/blob/09ecd17ab00c6f7960ec3b79a3ee8c206cef2c9c/tests/test_clients_GcpClient.py#L819), maybe instead of return_value you can use side_effect.